### PR TITLE
[WebSite]Information correction after graduation

### PR DESCRIPTION
**Pull request by QuakeWang**
Submit the pull request based on this [issue](https://github.com/apache/dolphinscheduler/issues/5295),

### CODE REVIEW

File: community/zh-cn/team.md [hunk 1]
```diff
@@ -1,76 +1,74 @@
...
 
 
 ## 贡献者
 
-DolphinScheduler有数百个贡献者，您可以在我们存储库的贡献列表中找到他们。
+DolphinScheduler 有数百个贡献者，您可以在我们存储库的贡献列表中找到他们。
 
 - [DolphinScheduler main repository](https://github.com/apache/incubator-dolphinscheduler/graphs/contributors)
 - [DolphinScheduler website](https://github.com/apache/incubator-dolphinscheduler-website/graphs/contributors)
 - [DolphinScheduler maven plugin](https://github.com/apache/incubator-dolphinscheduler-maven-plugin/graphs/contributors)
```
Comment (CalvinKirs) [inline community/zh-cn/team.md:63]: The warehouse address does not appear to have been modified

File: development/zh-cn/development-environment-setup.md [hunk 2]
```diff
@@ -1,44 +1,53 @@
 #### 准备工作
-1. 首先从远端仓库fork [dolphinscheduler](https://github.com/apache/incubator-dolphinscheduler) 一份代码到自己的仓库中
-2. 在开发环境中安装好MySQL/PostgreSQL、JDK、MAVEN
-3. 把自己仓库clone到本地
 
-    ` git clone https://github.com/apache/incubator-dolphinscheduler.git`
-    
-4. git clone项目后，进入项目目录，执行以下命令。
+1. 首先从远端仓库 fork [dolphinscheduler](https://github.com/apache/incubator-dolphinscheduler) 一份代码到自己的仓库中
```
Comment (CalvinKirs) [inline development/zh-cn/development-environment-setup.md:3]: The warehouse address does not appear to have been modified

**Comment by QuakeWang** [comment]
I have modified the warehouse address, please review the files

**Review by CalvinKirs — APPROVED** [review]
LGTM

**Pull request merged**